scripts/install_deps.py

```diff
@@ -144,9 +177,50 @@ def linux(self):
             linux.run_command(command_pre, check)
 
         command = self.config.get_os_deps_command(linux_distro=distro)
+        optional = self.config.get_os_deps_value(
+            "optional", linux_distro=distro, required=False
+        )
+        if optional:
+            for optional_package in optional:
```

Comment:
python trick:

```suggestion
            for optional_package in optional or []:
```

`None or []` resolves to `[]` so it's a nice shortcut for dealing with potential `None` values to avoid extra indention. Same below for subprojects